Packs/PrismaCloudCompute/Integrations/PaloAltoNetworks_PrismaCloudCompute/PaloAltoNetworks_PrismaCloudCompute.yml

```diff
@@ -1,79 +1,74 @@
-category: Network Security
```

Comment:
Revert all changes to this file besides your added command.

Reply:
Not sure why so many things got changed. I had to manually download the integration from the UI, then run demisto-sdk split, because the demisto-sdk download command wasn't working for me. Will try taking the existing YML file and manually pasting in the new part.